apps/humanatlas.io/src/app/components-v2/section-cards/section-cards.component.html

```diff
@@ -0,0 +1,14 @@
+<span class="section-title">Explore the Human Reference Atlas</span>
+
+<div class="cards">
+  @for (card of cardInfo(); track card) {
+    <!-- <a class="section-card" [routerLink]="['', card.route]" target="_blank" rel="noopener"> -->
+    <a class="section-card" target="_blank" rel="noopener">
+      <img class="card-image" [src]="card.imageSrc | assetUrl" [alt]="card.description" />
```

Comment:
Set `alt=""`. The image is purely decorative and the description is included below anyways